shavar/tests/test.ini

```diff
@@ -0,0 +1,164 @@
+###
+# app configuration
+# http://docs.pylonsproject.org/projects/pyramid/en/1.5-branch/narr/environment.html
+###
+
+[app:shavar]
+use = egg:shavar
+
+pyramid.reload_templates = true
+pyramid.debug_authorization = false
+pyramid.debug_notfound = false
+pyramid.debug_routematch = false
+pyramid.default_locale_name = en
+pyramid.includes =
+    pyramid_debugtoolbar
+
+# By default, the toolbar only appears for clients from IP addresses
+# '127.0.0.1' and '::1'.
+# debugtoolbar.hosts = 127.0.0.1 ::1
+
+[filter:translogger]
+use = egg:Paste#translogger
+setup_console_handler = False
+
+[pipeline:main]
+pipeline = translogger
+           shavar
+
+###
+# wsgi server configuration
+###
+
+[server:main]
+use = egg:waitress#main
+#host = 0.0.0.0
+host = 127.0.0.1
+port = 6543
+
+[uwsgi]
+http = 127.0.0.1:6543
+processes = 4
+threads = 2
+pythonpath = local/bin
+paste-logger = %p
+
+[loggers]
+keys = root, shavar
+
+[handlers]
+keys = console
+
+[formatters]
+keys = generic
+
+[logger_root]
+level = INFO
+handlers = console
+
+[logger_shavar]
+level = DEBUG
+handlers =
+qualname = shavar
+
+[handler_console]
+class = StreamHandler
+args = (sys.stderr,)
+level = NOTSET
+formatter = generic
+
+[formatter_generic]
+#format = %(asctime)s %(levelname)-5.5s [%(name)s][%(threadName)s] %(message)s
+# The following will replicate production logging styles:
+class = mozsvc.util.JsonLogFormatter
+
+[heka]
+logger = shavar
+severity = 4
+stream_class = heka.streams.UdpStream
+stream_host = localhost
+stream_port = 5565
+
+[heka_plugin_raven]
+provider = heka_raven.raven_plugin:config_plugin
+dsn = udp://username:password@localhost:9001/2
+override = True
+
+
+[shavar]
+# A newline separated list of the lists to be served.  The names given
+# here will be used to locate the list specific configuration stanzas
+# elsewhere in the INI file.
+lists_served = mozpub-track-digest256
+               moz-abp-shavar
```

Comment:
I just filed #79 for this since I imagine you copied this from somewhere else, but this is not a real list.